js_03-04/js/script.js

```diff
@@ -45,63 +35,123 @@ var pageObj = {
 
-        this.insertNewDomElement( '.container', domElements.div, ['content'] );
+        var objDiv = {
+            parent: ".container",
+            childTag: "div",
+            childClass: ['content']
+        };
+
+        this.insertNewDomElement( objDiv );
+
+        var objOl = {
+            parent: ".content",
+            childTag: "ol",
+            childClass: ['questions']
+        };
 
-        this.insertNewDomElement( '.content', domElements.ol, ['questions'] );
+        this.insertNewDomElement( objOl );
 
-        var propsLi = {};
+        var objLi = {
+            parent: ".questions",
+            childTag: "li",
+            childClass: "",
+            innerHTML: ""
+        };
 
         for ( var i = 1; i <= 3; i++ ) {
-            propsLi.innerHTML = "Вопрос №" + i;
-            this.insertNewDomElement( '.questions', domElements.li, '', propsLi );
+            objLi.innerHTML = "Вопрос №" + i;
+            this.insertNewDomElement( objLi );
         }
 
     },
 
     addAnswerCheckboxes: function() {
 
-        this.insertNewDomElement( '.questions li', domElements.ul, ['answers'] );
+        var objUl = {
+            parent: ".questions li",
+            childTag: "ul",
+            childClass: ['answers']
+        };
+
+        this.insertNewDomElement( objUl );
+        
+        var objLi = {
+            parent: ".answers",
+            childTag: "li",
+            childClass: ""
+        };
         
         for ( var j = 0; j < 3; j++ ) {
-            this.insertNewDomElement( '.answers', domElements.li, '' );
+            this.insertNewDomElement( objLi );
         }
 
-        var cssClassArray = ['custom-control', 'custom-checkbox'];
-        this.insertNewDomElement( '.answers li', domElements.div, cssClassArray );       
+        var objDiv = {
+            parent: ".answers li",
+            childTag: "div",
+            childClass: ['custom-control', 'custom-checkbox']
+        };
 
-        var inputProps = {
+        this.insertNewDomElement( objDiv );       
+
+        var objInput = {
+            parent: ".custom-checkbox",
+            childTag: "input",
+            childClass: ['custom-control-input'],         
             type: 'checkbox'
         };
 
-        this.insertNewDomElement( '.custom-checkbox', domElements.input, ['custom-control-input'], inputProps );
+        this.insertNewDomElement( objInput );
+
+        var objLabel = {
+            parent: ".custom-checkbox",
+            childTag: "label",
+            childClass: ['custom-control-label']
+        };
 
-        this.insertNewDomElement( '.custom-checkbox', domElements.label, ['custom-control-label'] );
+        this.insertNewDomElement( objLabel );
 
     },
 
     addButton: function() {
 
-        this.insertNewDomElement( '.content', domElements.div, ['submit-btn'] );
+        var objDiv = {
+            parent: ".content",
+            childTag: "div",
+            childClass: ['submit-btn']
```

Comment:
please use `'` or `"` everywhere to keep the code in one style
also, it's a commons practice to use single quotes `'` in `js` files

Reply:
Done. 
For now, there are everywhere single quotes.